privacy/use-cases/src.md

```diff
@@ -0,0 +1,28 @@
+# EMV&reg; Secure Remote Commerce (SRC) User Recognition
+<sup>[Home][home] > [Use Cases][use-cases] > SRC User Recognition</sup>
+
+See more [information about EMV&reg;SRC](https://www.emvco.com/emv-technologies/src/), including the "Click-to-Pay" consumer facing UX.
+
+## Overview of Click-to-Pay Flow
+
+* The user pushes the Click-to-Pay button.
```

Comment:
Optional; should we indicate that the user is usually **not** on an SRC-I page here? (As otherwise step 2 is just a 1p cookie)